deployments/tike/environments/tess/tests/imports

```diff
@@ -8,15 +8,13 @@ numpy
 numba
 pandas
 sklearn
-s3fs
+###s3fs
```

Comment:
Why are we removing this check? I see s3fs added above

Reply:
Good catch,  I'm winging it some here,  I think formally it is no longer a requirement but it is required for some of the webinar notebooks so I added it back informally to keep moving.